ui-components/src/forms/DateField.tsx

```diff
@@ -4,49 +4,37 @@ import { Field } from "./Field"
 import moment from "moment"
 import { UseFormMethods, FieldError, DeepMap } from "react-hook-form"
 
-export type DOBFieldValues = {
-  birthDay: string
-  birthMonth: string
-  birthYear: string
+export type DateFieldValues = {
+  day: string
+  month: string
+  year: string
 }
-export interface DOBFieldProps {
-  error?: DeepMap<DOBFieldValues, FieldError>
+export interface DateFieldProps {
+  error?: DeepMap<DateFieldValues, FieldError>
   errorMessage?: string
   label: React.ReactNode
   register: UseFormMethods["register"]
   watch: UseFormMethods["watch"]
-  defaultDOB?: DOBFieldValues
-  atAge?: boolean
-  validateHhAge?: boolean
+  defaultDate?: DateFieldValues
   name?: string
   id?: string
   required?: boolean
   disabled?: boolean
   readerOnly?: boolean
+  validateAge18?: boolean
 }
 
-const DOBField = (props: DOBFieldProps) => {
-  const { defaultDOB, error, register, watch, atAge, validateHhAge, name, id, errorMessage } = props
+const DateField = (props: DateFieldProps) => {
+  const { defaultDate, error, register, watch, name, id, errorMessage } = props
   const fieldName = (baseName: string) => {
     return [name, baseName].filter((item) => item).join(".")
```

Comment:
Hey I know this isn't part of the code you touched, but initially the term `fieldName` threw me (wasn't sure what it's for)…and even looking at the code now it's hard for me to initially grok why it's there and what it's doing. Any thoughts on refactoring and/or adding some comments?

Reply:
Sure!